src/containers/mobile-connect-wallets-drawer/MobileConnectWalletsDrawer.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+import { getIsOpen } from './store/selectors'
+import { getKeyboardVisibility } from 'store/application/ui/mobileKeyboard/selectors'
+
+import Drawer from 'components/drawer/Drawer'
+import styles from './MobileConnectWalletsDrawer.module.css'
+
+const messages = {
+  title: 'Connect Additional Wallets With Your Account',
+  visit: 'Visit audius.co from a desktop browser',
+  reason:
+    'Show off your NFT Collectibles and flaunt your $AUDIO with a VIP badge on your profile.'
+}
+
+const MobileConnectWalletsDrawer = ({ onClose }: { onClose: () => void }) => {
```

Comment:
just a stylistic thing, for these I've been creating the onClose action creator inside the component rather than passing it by props, since we're already connected to the store to get the visibility state. how do you feel about that vs passing it in?

Reply:
I could go either way. In this case the parent has to open the modal which is pretty similar to the close function so I don't necessary think it's cleaner either way.